packages/legacy/src/Plans/components/Wizard/TypeForm.tsx

```diff
@@ -60,8 +62,9 @@ export const TypeForm: React.FunctionComponent<ITypeFormProps> = ({
               isInline
               title="Warm migration is not currently available."
             >
-              Warm migration from {PROVIDER_TYPE_NAMES.ovirt} source provider is only supported
-              with a verified secure connection.
+              Warm migrations from {PROVIDER_TYPE_NAMES.ovirt} source providers are only supported
+              with a verified secure connection.  To enable a warm migration, first select a secure provider
```

Comment:
a user want to select this vm on this provider, selecting a different provider can't help here

Reply:
another option is to sent the user to the documentation to get more information, but this will require to import from newer parts into the legacy code.

Reply:
Yeah there really isn't a great option here. I updated back to a minimal message.